distributed/comm/ucx.py

```diff
@@ -18,11 +17,11 @@
 from .utils import ensure_concrete_host, to_frames, from_frames
 from ..utils import ensure_ip, get_ip, get_ipv6, nbytes, log_errors
 
+import dask
+import ucp
+import numpy as np
 
-os.environ.setdefault("UCX_RNDV_SCHEME", "put_zcopy")
-os.environ.setdefault("UCX_MEMTYPE_CACHE", "n")
-os.environ.setdefault("UCX_TLS", "all")
-os.environ.setdefault("UCX_SOCKADDR_TLS_PRIORITY", "sockcm")
```

Comment:
Do we still want to set these configuration values?  If so, we need to add them to the `distributed.yaml` file below.

When running on my linux laptop though I don't need to set them, and I have a preference to avoid setting them if they are not necessary.

cc @pentschev @quasiben

Reply:
The `ucp.init` call just below fully replaces this, so we don't need them any longer.

Reply:
Great.  @pentschev quick check.  Do you agree with the changes in this PR?  Should it be merged?

Reply:
Yes, we should merge it.